label_studio/ml_models/models.py

```diff
@@ -75,3 +76,52 @@ class ThirdPartyModelVersion(ModelVersion):
 
     def has_permission(self, user):
         return user.active_organization == self.organization
+
+
+class ModelRun(models.Model):
+    class ProjectSubset(models.TextChoices):
+        ALL = 'All', _('All')
+        HASGT = 'HasGT', _('HasGT')
+
+    class FileType(models.TextChoices):
+        INPUT = 'Input', _('Input')
+        OUTPUT = 'Output', _('Output')
+
+    class ProjectStatus(models.TextChoices):
+        PENDING = 'Pending', _('Pending')
+        COMPLETED = 'Completed', ('Completed')
+        FAILED = 'Failed', ('Failed')
+        CANCELED = 'CANCELED', ('CANCELED')
+
+    organization = models.ForeignKey(
+        'organizations.Organization', on_delete=models.CASCADE, related_name='model_runs', null=True
+    )
+
+    project = models.ForeignKey(Project, on_delete=models.CASCADE, related_name='model_runs')
+
+    model_version = models.ForeignKey(ThirdPartyModelVersion, on_delete=models.CASCADE, related_name='model_runs')
+
+    created_by = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        related_name='model_runs',
+        on_delete=models.SET_NULL,
+        null=True,
+    )
+
+    project_subset = models.CharField(max_length=255, choices=ProjectSubset.choices, default=ProjectSubset.HASGT)
+    status = models.CharField(max_length=255, choices=ProjectStatus.choices, default=ProjectStatus.PENDING)
+
+    created_at = models.DateTimeField(_('created at'), auto_now_add=True)
```

Comment:
it would be helpful to have updated_at too here

Reply:
@makseq - are you thinking about metrics gathering for figuring out how long a run stays in particular state? @pakelley mentioned it is ok not having it in comment above.

Reply:
@makseq yeah, if there's a good reason to add it in this case I'm up for it, but since this object has a particularly well-defined set of states, I'd rather keep track of updates via the `triggered_at` and `completed_at` attributes. These are the only times the object should be updated.

Reply:
@makseq does this seem good to you? And if so, mind giving us an approval here?